en/snap/with-plugins.md

```diff
@@ -420,268 +494,352 @@ Customer receives email notification| Receive notification|  Check MAP menu sett
 
-1. Perform successful transaction in your online store by inputing a dummy credit card number below (Sandbox Mode, [other test credentials](/en/technical-reference/sandbox-test.md)):
-  * **Card Number**: 4811 1111 1111 1114
-  * **CVV**: 123
-  * **Exp. Month**: 01
-  * **Exp. Year**: 2025
+| Name        | Value               |
+| :---------- | :------------------ |
+| Card Number | 4811 1111 1111 1114 |
+| CVV         | 123                 |
+| Exp Month   | 01                  |
+| Exp Year    | 2025                |
+| OTP/3DS     | 112233              |
 
+For more details, refer to [Testing Payments on Sandbox](/en/technical-reference/sandbox-test).
 2. Examine a few points below to ensure plugin is installed and performs properly.
 
-Check Point| Expected Result |When Unexpected Result Occurs
---- | --- | ---
-Check order status in CMS back end|  Order status recorded in backend| Check endpoint/payment notification URL setting in MAP.<br> Check if your CMS/notification url can be accessed from public internet properly
-Merchant receives email notification|  Receive notification|  Check MAP menu setting - email notification
-Customer receives email notification| Receive notification|  Check MAP menu setting - email notification
+| Check Point                 | Error                                     | Troubleshooting                                              |
+| --------------------------- | ----------------------------------------- | ------------------------------------------------------------ |
+| Order status in CMS backend | Order status not recorded in the backend. | Check endpoint/Payment Notification URL setting on MAP.<br> Check if your CMS/Notification URL is publicly accessible. |
+| Merchant email notification | Notification not received.                | Check Email Notifications settings on MAP. For more details, refer to [Configuring Email Notifications](/en/after-payment/dashboard-usage.md#configuring-email-notifications). |
+| Customer email notification | Notification not received.                | Check Email Notifications settings on MAP. For more details, refer to [Configuring Email Notifications](/en/after-payment/dashboard-usage.md#configuring-email-notifications). |
 
 #### Payment Example
 ![Drupal Payment Test](./../../asset/image/drupal8-pay-show.gif)
 
 </article>
 </details>
+
 <hr><br><br>
 
-### Wordpress - Easy Digital Download
+### WordPress - Easy Digital Download
 <hr>
 
-Midtrans ❤️ EDD! Integrate your Easy Digital Download store with Midtrans Snap payment gateway. We strive to make payments simple for both the merchant and customers. This plugin will allow online payment on your EDD store using various online payment channel. 
+Midtrans ❤️ EDD! Integrate your Easy Digital Download (EDD) store with Midtrans Snap payment gateway. Midtrans makes payments simple for both the merchant and the customers. This plugin will allow online payments on your EDD store using various online payment channel.
```

Comment:
check the comment on the statement with the word "strive" given above. Check globally.

Reply:
various channel or various channels ?